modal/functions.py

```diff
@@ -924,7 +925,7 @@ def from_parametrized(
         obj,
         from_other_workspace: bool,
         options: Optional[api_pb2.FunctionOptions],
-        args: Iterable[Any],
+        args: Sized,
```

Comment:
I haven't heard of this before, what is the difference between `Iterable[Any]` -> `Sized`?

Reply:
`Iterable` supports any object that can be iterated over (implements `__iter__`) vs `Sized` supports any object that has a length (implements `__len__`). Generator can be iterated over but will fail this line [here](https://github.com/modal-labs/modal-client/pull/1328/commits/16ba69a4fa0a782df9410ebe135531296a44c69c#diff-87e1f4fd05c5e1972a1b8d781a518822704dd8fba567111f9b74ad8a3c9959fdR949).